metadata-ingestion/docs/transformer/dataset_transformer.md

```diff
@@ -668,6 +668,56 @@ We can add glossary terms to datasets based on a regex filter.
               ".*example1.*": ["urn:li:glossaryTerm:Email", "urn:li:glossaryTerm:Address"]
               ".*example2.*": ["urn:li:glossaryTerm:PostalCode"]
     ```
+
+## Tags to Term Mapping
+### Config Details
+
+| Field         | Required | Type               | Default     | Description                                                                                           |
+|---------------|----------|--------------------|-------------|-------------------------------------------------------------------------------------------------------|
+| `tags`        | ✅       | List[str]          |             | List of tag names based on which terms will be created and associated with the dataset.               |
+| `semantics`   |          | enum               | "OVERWRITE" | Determines whether to OVERWRITE or PATCH the terms associated with the dataset on DataHub GMS.        |
+
+<br/>
+
+The `tags_to_term` transformer is designed to map specific tags to glossary terms within DataHub. It takes a configuration of tags should be translated into corresponding glossaryTerm. This transformer can apply these mappings to any tags found either at column level of dataset or dataset top level.
+
+When specifying tags in the configuration, use the tag's simple name rather than the full tag URN.
+
+For example, instead of using the tag URN `urn:li:tag:snowflakedb.snowflakeschema.tag_name:tag_value`, you should specify just the tag name `tag_name` in the mapping configuration
+
+```yaml
+transformers:
+  - type: "tags_to_term"
+    config:
+      semantics: OVERWRITE  # OVERWRITE is the default behavior
+      tags:
+        - "tag_name"
```

Comment:
Just want to make sure I'm understanding correctly - if there's a tag called "tag_name", this will create + map to a term called "tag_name"? Is that right?

Reply:
hi @eboneil yes in snowflake we have tag and its value, we will be creating term with that tag name.